web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -72,20 +65,55 @@ export class ShardComponent implements OnInit, OnDestroy {
     });
   }
 
+  getActions() {
+    let result: MenuItem[] = [
+      {label: 'Status', items: [
+        {label: 'Validate', command: (event) => {this.openValidateShardDialog(); }},
+        {label: 'Validate Version', command: (event) => {this.openValidateVerShardDialog(); }},
```

Comment:
Sorry, github treated my \<something\> as html tag and didn't show it to you. So my suggestion was to add something after word "Validate", so that it was e.g. "Validate Shard" and "Validate Version" or something like that.